### ISSUE TITLE
SVG rendering some artefacts in 1.1.1 but not in 1.1.0

### ISSUE THREAD

**Issue by ycherniavskyi** [dnfield/flutter_svg#728]
After update to 1.1.1 odd strokes rendered (it is not rendered in any other viewers and event in 1.1.0).

Here is the svg (I also try optimised version after [svgomg](https://jakearchibald.github.io/svgomg/) -  result is the same):
```xml
<?xml version="1.0" encoding="UTF-8" standalone="no"?>
<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
<svg width="100%" height="100%" viewBox="0 0 100 100" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" xml:space="preserve" xmlns:serif="http://www.serif.com/" style="fill-rule:evenodd;clip-rule:evenodd;stroke-linejoin:round;stroke-miterlimit:2;">
    <g transform="matrix(17.1087,0,0,2.04082,-20051.4,-500)">
        <g id="Logo">
            <rect x="1172" y="245" width="46" height="49" style="fill:none;"/>
            <g transform="matrix(0.0584498,0,0,0.49,1138.27,-481.18)">
                <g>
                    <g id="mini-logo" serif:id="mini logo">
                        <g transform="matrix(2.32558,0,0,2.32558,577,1482)">
                            <path d="M0,40C0,41.7 1.3,43 3,43L40,43C41.7,43 43,41.7 43,40L43,3C43,1.3 41.7,0 40,0L3,0C1.3,0 0,1.3 0,3L0,40Z" style="fill:rgb(1,158,248);fill-rule:nonzero;"/>
                        </g>
                        <g transform="matrix(2.32558,0,0,2.32558,577,1482)">
                            <path d="M24.4,32L24.4,11C24.4,9.9 24,9.5 23,9.6L22.5,9.6C21.3,9.5 20.3,9.6 19.9,11.1C19.6,12.2 18.7,13 17.4,13.2C16.6,13.3 15.8,13.5 15,13.6C14.3,13.6 14,13.9 14,14.7L14,15.9C13.9,16.8 14.3,17.1 15.1,17C16.1,16.9 17.1,17 18.1,17C19.2,16.9 19.5,17.3 19.5,18.4C19.4,22.3 19.4,26.1 19.5,30C19.6,34.1 18.6,33.3 23,33.4C24.1,33.4 24.4,33 24.4,32Z" style="fill:white;fill-rule:nonzero;"/>
                        </g>
                    </g>
                </g>
            </g>
        </g>
    </g>
</svg>
```

Rendering in 1.1.1:
![image](https://user-images.githubusercontent.com/552025/176430841-626468c5-f178-4883-8488-20d7e8a49e25.png)

Rendering in 1.1.0:
![image](https://user-images.githubusercontent.com/552025/176430874-ec9e2778-7109-4c05-a34d-77eb99631456.png)

It seems that the reason in `stroke-linejoin:round`, but if it is ok for all other viewers then it seems should be ok for [flutter_svg](https://github.com/dnfield/flutter_svg) 🤷🏻‍♂️.

**Comment by ycherniavskyi** [comment]
@iBog no, rollback to previous version fore sure help, make sure that you also rollback version in `pubspec.lock` file.

**Comment by iBog** [comment]
> @iBog no, rollback to previous version fore sure help, make sure that you also rollback version in `pubspec.lock` file.

you are right, pubspec.lock still used 1.1.1

**Comment by iBog** [comment]
Rendering issue in my project
![svg_bug](https://user-images.githubusercontent.com/168304/176846442-e7f8c9e4-daf8-4847-888a-0837bf37d20b.jpg)

**Comment by dnfield** [comment]
Yup this is a regression I introduced when fixing a different bug. I'll look into it.

**Comment by dnfield** [comment]
Published 1.1.1+1, and retracted 1.1.1

**Comment by iBog** [comment]
for me ver. 1.1.1+1 has fixed problem, thanks!
